conf/config-default.yaml

```diff
@@ -114,6 +114,10 @@ apisix:
     key_encrypt_salt: "edd1c9f0985e76a2"    #  If not set, will save origin ssl key into etcd.
                                             #  If set this, must be a string of length 16. And it will encrypt ssl key with AES-128-CBC
                                             #  !!! So do not change it after saving your ssl, it can't decrypt the ssl keys have be saved if you change !!
+  enable_control: true
```

Comment:
I think the default value should be`false`

Reply:
keep current way

Reply:
Agree, and we need more comments here.